optaplanner-core/src/main/java/org/optaplanner/core/api/solver/SolverFactory.java

```diff
@@ -177,20 +177,6 @@
         return new DefaultSolverFactory<>(solverConfig);
     }
 
-    /**
-     * As defined by {@link #create(SolverConfig)}.
-     * @param solverConfig never null
-     * @param classLoader sometimes null, the {@link ClassLoader} to use for loading all resources and {@link Class}es,
-     *      null to use the default {@link ClassLoader}
-     * @return never null
-     * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
-     */
-    public static <Solution_> SolverFactory<Solution_> create(SolverConfig solverConfig, ClassLoader classLoader) {
-        // Defensive copy of solverConfig, because the DefaultSolverFactory doesn't internalize it yet
-        solverConfig = new SolverConfig(solverConfig);
-        return new DefaultSolverFactory<>(solverConfig, new SolverConfigContext(classLoader));
-    }
```

Comment:
Again, removing public API.

Reply:
If it wasn't released yet, then Revapi wouldn't catch it and therefore wouldn't fail. I'm not in VPN right now, so cannot check why specifically the build failed.

Reply:
@yurloc is right, revapi complains about removal of org.optaplanner.core.config.SolverConfigContext::getClassLoader()

Reply:
SolverConfigContext is in the config package. So it's special :)
It's not in the api package, nor in the impl package.
It's in the package for which we're guarantee BC only from the XML point of view not from the methods point of view. The lack of a clean line is hurting us and should be dealt with in the 8.0 timeframe.